src/components/DropdownWithKebab.tsx

```diff
@@ -38,11 +38,10 @@ const DropdownWithKebab: React.FC<DropdownWithKebabProps> = ({ obj }) => {
     }
   };
 
-
   const onEditClick = () => {
     history.push({
-      pathname: `/k8s/ns/${obj.metadata.namespace}/tlspolicy/name/${obj.metadata.name}/edit`,
-    })
+      pathname: `/k8s/ns/${obj.metadata.namespace}/${obj.apiVersion.replace("/", "~")}~${obj.kind}/${obj.metadata.name}/yaml`,
```

Comment:
this breaks the edit for the rest if the policies. I have a change currently for DNS i you want you can leave it with me and ill have it that it works for auth to

Reply:
/yaml is the default yaml view for all openshift resources so our view would override them which is grand for auth but no bueno for the others since they're forms

Reply:
/sold